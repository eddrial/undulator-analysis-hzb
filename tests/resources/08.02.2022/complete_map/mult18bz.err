 error report for the data set mult18bz.dvm        
--------------------------------------------
      
         127   data points thrown away to get a
         monotonically increasing data set
 the truncated data set has only   1591 data points 
        out of totally   2558 data points
 the derivative at  0.42936E+03 and  0.45429E+03 is zero 
 the derivative at  0.50880E+03 and  0.54006E+03 is zero 
 the derivative at  0.64950E+03 and  0.68035E+03 is zero 
 the derivative at  0.68035E+03 and  0.70485E+03 is zero 
 the derivative at  0.98711E+03 and  0.10404E+04 is zero 
 the derivative at  0.11223E+04 and  0.11532E+04 is zero 
 the derivative at  0.12385E+04 and  0.13180E+04 is zero 
 the derivative at  0.13180E+04 and  0.13497E+04 is zero 
 the derivative at  0.14084E+04 and  0.14329E+04 is zero 
 the derivative at  0.14329E+04 and  0.14642E+04 is zero 
 the derivative at  0.16852E+04 and  0.17173E+04 is zero 
 the derivative at  0.17173E+04 and  0.17422E+04 is zero 
 the derivative at  0.17975E+04 and  0.18487E+04 is zero 
 the derivative at  0.18487E+04 and  0.18825E+04 is zero 
 the derivative at  0.20481E+04 and  0.20807E+04 is zero 
 the rms value of the residuals is  0.24508E-02
 the maximum residual is  0.10024E-01
 the rms value of the lambda/2-filtered data is  0.14571E-02
 the maximum value of the lambda/2-filtered data is  0.73759E-02
