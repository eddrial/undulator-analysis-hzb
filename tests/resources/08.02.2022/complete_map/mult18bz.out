mult18bz.dvm
            

 names of parameters as follows : 
b(1) == amplitude, 1st harm.
b(2) == amplitude, 3rd harm.
b(3) == amplitude, 5rd harm.
....                        
b(n) == amplitude, n^th harm.
b(n+1) == lambda, mm.
b(n+2) == x0, 1st harm, mm.
b(n+3) == x0, 3rd harm, mm.
b(n+4) == x0, 5th harm, mm.
....
b(nparm) == x0, n^th harm, mm.

 parameters:
   1 -8.9150118E-03
   2  1.0898515E-04
   3  1.1875838E-04
   4  8.9152336E-06
   5 -2.5067727E-06
   6  5.5992882E+01
   7  1.0421499E+03
   8  1.0424958E+03
   9  1.0449330E+03
  10 -1.2799597E-01
  11  2.1812084E+00
 number of iterations:   12
     sumsq:  9.55644E-03
       rms:  2.45083E-03  dimension of magnetic field
       rms:  2.74907E+01  percent of b_eff
   rms_ext:  2.02036E+01  percent of b_eff
 B_eff =   8.9151E-03
 B_max =  -8.8098E-03
 K_eff =    0.047
 lamda_1 (nm) =      2.54304

        eigenvectors in parameter space

 error                         direction
    34.43,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
    12.45,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
     2.38,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
     1.31,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
     0.10,  0.00  0.00  0.00  0.00  0.00 -0.05  1.00  0.00  0.00  0.00  0.00
     0.01,  0.00  0.00  0.00  0.00  0.00  1.00  0.05  0.00  0.00  0.00  0.00
     0.00, -0.52  0.10  0.28  0.62  0.51  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.66  0.64  0.22  0.31  0.06  0.00  0.00  0.00  0.00  0.00  0.00
     0.00, -0.39  0.35  0.63 -0.25 -0.52  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.22 -0.24  0.55 -0.50  0.58  0.00  0.00  0.00  0.00  0.00  0.00
     0.00, -0.29  0.63 -0.41 -0.46  0.37  0.00  0.00  0.00  0.00  0.00  0.00
   
  calculated kicks for trajectory compensation: 
 kick_1:  -0.08641 Tmm
 kick_2:   0.08641 Tmm
      
 the rms value of the second derivative is  0.00000E+00
 the maximum value of the second derivative is  0.00000E+00
      
 the maximum transverse movement of the electron in mu_meter
                  (peak to peak) is  0.38233E+01
      
 the truncated data set has only   1591 data points 
      
 the rms value of the residuals is  0.24508E-02
 the maximum residual is  0.10024E-01
      
 the rms value of the lambda/2-filtered data is  0.14571E-02
 the maximum value of the lambda/2-filtered data is  0.73759E-02
      
 the rms phase error (all data) is    0.798 deg.
 the maximum phase error (all data) is    3.819 deg.
      
 the rms phase error (extrema) is    3.289 deg.
 the maximum phase error (extrema) is    7.344 deg.
      
 the Walker-rms phase error (all data) is    0.799 deg.
 the Walker-maximum phase error (all data) is    3.892 deg.
      
 the Walker-rms phase error (extrema) is    0.776 deg.
 the Walker-maximum phase error (extrema) is    1.973 deg.
  
  phase: straight line subtracted
 -----------------------------------------
 total phase error            0.7978833    
 local phase error            0.3567664    
 slowly varying phase error   0.7132043    
 oscillatory phase error      2.5981901E-02
 -----------------------------------------
